v20.2/build-a-go-app-with-cockroachdb.md

```diff
@@ -36,152 +32,57 @@ To install the [CockroachDB Go library](https://pkg.go.dev/github.com/cockroachd
 $ go get -u github.com/cockroachdb/cockroach-go/crdb
 ~~~
 
-<section class="filter-content" markdown="1" data-scope="secure">
-
-## Step 3. Create the `maxroach` user and `bank` database
+## Step 3. Start CockroachDB
```

Comment:
I believe running `go mod init` below eliminates the need for Steps 1 & 2. All the library deps are resolved at that point.